### ISSUE TITLE
Getting error while evaluating NER model

### ISSUE THREAD

**Issue by ambuje** [flairNLP/flair#3014]
Hi Team,

Having issues while evaluating a NER model 

**Code for evaluating: -**
model = SequenceTagger.load('resources/taggers/example-ner4/best-model.pt')
result = model.evaluate(corpus.test, gold_label_type="ner", mini_batch_size=8)

**While training the model the Sequence Tagger:-** 
from flair.models import SequenceTagger

tagger : SequenceTagger = SequenceTagger(hidden_size=256,
                                       embeddings=embeddings,dropout=0.1,locked_dropout=0.1,rnn_type='LSTM',
                                       tag_dictionary=tag_ditionary1,train_initial_hidden_state=False,
                                       tag_type=tag_type,rnn_layers=4,tag_format='BIO',
                                       use_crf=False)

**Error:-** 
The string 'E-ORG' is not in dictionary! Dictionary contains only: ['O', 'B-PERSON', 'I-PERSON', 'B-ORG', 'I-ORG', 'B-GPE', 'I-GPE', 'B-EMAIL', 'I-EMAIL']
2022-12-07 14:04:32,030 You can create a Dictionary that handles unknown items with an <unk>-key by setting add_unk = True in the construction.

**Comment by helpmefindaname** [comment]
Hi @ambuje 
I suppose you don't want to share the model, but can you please provide the output of the following code:

```
import torch
from flair.models import SequenceTagger

path = 'resources/taggers/example-ner4/best-model.pt'
model = SequenceTagger.load(path)

print("Tagger - tag-format: " + model.tag_format)

state_dict = torch.load(path)
print("State dict - tag-format: " + state_dict.get("tag_format"))
```

I suppose that either the `tag_format` is not set (and defaults to BIOES in that case) or is set to BIOES for some reason.

If that is the case, you can solve your issue by using:
```
from flair.models import SequenceTagger

path = 'resources/taggers/example-ner4/best-model.pt'
model = SequenceTagger.load(path)
model.tag_format = "BIO"
model.save(path)
```
